tests/collections/test_collections.py

```diff
@@ -59,7 +59,7 @@ def test_mapping_over_observables(self):
         obs_list = [qml.PauliX(0), qml.PauliY(0)]
         template = lambda x, wires: qml.RX(x, wires=0)
 
-        qc = qml.map(template, obs_list, dev)
+        qc = qml.map(template, obs_list, dev, diff_method="parameter-shift")
```

Comment:
Why is the parameter shift needed here to get the tests to pass? :thinking:

Reply:
This is because of the assertion:

```python
assert qc[0].device is qc[1].device
```

🤔

Reply:
Oh, true, because the device is changed midway when using backprop. This seems a bit weird, and confusing to be honest. It has more to do with the way a device/child-device is chosen rather than that the parameter-shift method needs to be used. I feel like these tests perhaps shouldn't check that the device stays the same any more, since this i obviously not the case, rather than locking it to a specific diff method. :thinking:

Reply:
Good call, maybe I will just remove the device assertions.

Reply:
Have removed these :)